lights/lights.c

```diff
@@ -125,6 +128,10 @@ static int set_light_backlight(struct light_device_t *dev,
     return -ENXIO;
   }
 
+  if (max_brightness == -1) {
+    return -EINVAL;
```

Comment:
correct the Indentation

Reply:
it is done by taking reference of other if statements in code.